lab/dp/problems/sum_query/enunt.md

```diff
@@ -0,0 +1,27 @@
+# Gigel si suma elementelor din matrice
+
+*Gigel* s-a lasat de stropitori si de mancat pufuleti, asa ca s-a apucat de 
+carte si a inceput fix cu ce era mai greu. Asa ca el va cere voua ajutorul,
+ca sa poate spune prietenilor lui ca e destept. Gigel va da o matrice N * M,
+cu numere intregi. El se intreaba cum poate calcula extrem de rapid (cu viteza
+tunetului si a fulgerului) suma dintr-un patrat oarecare din matrice (determinat
+de coordonatele initiale xi si yi, si coordonatele finale xf si yf). El 
```

Comment:
I thinl `initiale` and `finale` are not proper terms. We should use "stanga sus" si "dreapta jos" (top-left and bottom-right)